graal/backends/core/analyzers/scancode.py

```diff
@@ -22,38 +22,45 @@
 
 import json
 import subprocess
-
 from graal.graal import (GraalError,
                          GraalRepository)
 from .analyzer import Analyzer
 
 
+SCANCODE_CLI_EXEC = "etc/scripts/scancli.py"
+CONFIGURE_EXEC = 'configure'
+
+
 class ScanCode(Analyzer):
     """A wrapper for nexB/scancode-toolkit.
 
     This class allows to call scancode-toolkit over a file, parses
     the result of the analysis and returns it as a dict.
 
     :param exec_path: path of the scancode executable
+    :param cli: True, if scancode_cli is used
     """
-    version = '0.1.0'
+    version = '0.2.0'
 
-    def __init__(self, exec_path):
+    def __init__(self, exec_path, cli=False):
         if not GraalRepository.exists(exec_path):
             raise GraalError(cause="executable path %s not valid" % exec_path)
 
         self.exec_path = exec_path
+        self.cli = cli
 
-    def analyze(self, **kwargs):
-        """Add information about license
+        if self.cli:
+            exec_path = self.exec_path.replace(SCANCODE_CLI_EXEC, CONFIGURE_EXEC)
+            _ = subprocess.check_output([exec_path, '--help']).decode("utf-8")
```

Comment:
it shouldn't be `_ = subprocess.check_output([exec_path])` (as reported here: https://github.com/nexB/scancode-toolkit/blob/develop/etc/scripts/scanserv.README#L14) ?

Reply:
I guess the param `--help` runs ./configure under the hood. Thank you for updating it.